cipher/rot13.js

```diff
@@ -0,0 +1,19 @@
+// ROT13, adalah algoritme enkripsi sederhana yang menggunakan sandi abjad-tunggal dengan pergeseran k=13.
+// Enkripsi ini merupakan penggunaan dari sandi Caesar dengan geseran 13.
+// ROT13 biasanya digunakan di forum internet, agar spoiler, jawaban teka-teki, kata-kata kotor, dan
+// semacamnya tidak terbaca dengan sekilas.
+
+// normal function
+function rot13(str) {
+    return str.replace(/[a-zA-Z]/g, function(chr) {
+      var start = chr <= 'Z' ? 65 : 97;
```

Comment:
```suggestion
      var start = chr <= "Z" ? 65 : 97;
```
menggunakan tanda petik dua